packages/flutter_test/lib/src/window.dart

```diff
@@ -70,43 +73,65 @@ class TestWindow implements Window {
   @override
   Size get physicalSize => _physicalSizeTestValue ?? _window.physicalSize;
   Size _physicalSizeTestValue;
-  /// Hides the real physical size and reports the given [physicalSizeTestValue] instead.
+  /// Hides the real physical size and reports the given [physicalSizeTestValue]
+  /// instead.
   set physicalSizeTestValue (Size physicalSizeTestValue) {
     _physicalSizeTestValue = physicalSizeTestValue;
     onMetricsChanged();
   }
-  /// Deletes any existing test physical size and returns to using the real physical size.
+  /// Deletes any existing test physical size and returns to using the real
+  /// physical size.
   void clearPhysicalSizeTestValue() {
     _physicalSizeTestValue = null;
     onMetricsChanged();
   }
 
+  @override
+  double get physicalDepth => _physicalDepthTestValue ?? _window.physicalDepth;
+  double _physicalDepthTestValue;
+  /// Hides the real physical depth and reports the given
+  /// [physicalDepthTestValue] instead.
+  set physicalDepthTestValue (double physicalDepthTestValue) {
+    _physicalDepthTestValue = physicalDepthTestValue;
+    onMetricsChanged();
+  }
+  /// Deletes any existing test physical depth and returns to using the real
+  /// physical depth.
+  void clearPhysicalDepthTestValue() {
+    _physicalDepthTestValue = null;
+    onMetricsChanged();
+  }
+
   @override
   WindowPadding get viewInsets => _viewInsetsTestValue ??  _window.viewInsets;
   WindowPadding _viewInsetsTestValue;
-  /// Hides the real view insets and reports the given [viewInsetsTestValue] instead.
+  /// Hides the real view insets and reports the given [viewInsetsTestValue]
+  /// instead.
   set viewInsetsTestValue(WindowPadding viewInsetsTestValue) {
     _viewInsetsTestValue = viewInsetsTestValue;
     onMetricsChanged();
   }
-  /// Deletes any existing test view insets and returns to using the real view insets.
+  /// Deletes any existing test view insets and returns to using the real view
+  /// insets.
   void clearViewInsetsTestValue() {
     _viewInsetsTestValue = null;
     onMetricsChanged();
   }
 
-  // TODO(dnfield): Remove this ignore once custom embedders have had time to catch up
-  // And make this property actually wrap _window.viewPadding.
+  // TODO(dnfield): Remove this ignore once custom embedders have had time to
+  //  catch up and make this property actually wrap _window.viewPadding.
   // @override
   // ignore: annotate_overrides, public_member_api_docs
```

Comment:
Can we remove the TODO and the ignore here, and uncomment the override?  This should be all set.

Reply:
Sure thing!